.github/workflows/build.yml

```
@@ -57,11 +59,11 @@ jobs:
       - name: Deploy Production
         if: github.event_name == 'push'
         id: deploy-production
         run: |
-          bunx vercel pull --yes --environment=production --token=${{ secrets.VERCEL_TOKEN }}
-          bunx vercel build --prod --token=${{ secrets.VERCEL_TOKEN }}
-          bunx vercel deploy --prebuilt --prod --token=${{ secrets.VERCEL_TOKEN }}
\ No newline at end of file
+          npx vercel pull --yes --environment=production --token=${{ secrets.VERCEL_TOKEN }}
+          npx vercel build --prod --token=${{ secrets.VERCEL_TOKEN }}
+          npx vercel deploy --prebuilt --prod --token=${{ secrets.VERCEL_TOKEN }}
+hjelelel
```

actionlint (1.7.4)

could not parse as YAML: yaml: line 69: could not find expected ':'

(syntax-check)